qdrant-landing/content/documentation/cloud/backups.md

```diff
@@ -1,36 +1,95 @@
+    resets to the previous configuration. You may lose data after the date of 
+    that restored snapshot.
+
+### Configure a backup
+
+After you have taken the prerequisite steps, you can configure a backup with the
+[Qdrant Cloud Dashboard](https://cloud.qdrant.io). To do so, take these steps:
+
+1. Sign in to the dashboard
+1. Select Clusters.
+1. Select the cluster that you want to back up.
+   ![Select a cluster](/documentation/cloud/select-cluster.png)
+1. Find and select the **Backups** tab.
+1. Now you can set up a backup schedule.
+   The **Days of Retention** is the number of days after a backup snapshot is
+   deleted.
+1. Alternatively, you can select **Backup now** to take an immediate snapshot.
 
-To restore a Qdrant cluster from backup, you can select a desired backup copy version and start the restore process.
-Attention: during the restoring process the affected cluster will not be available because the cluster will be deleted and created from scratch from the backup copy.
-Please also note, that if you changed the cluster setup after the copy was created, the new cluster will reset to the previous configuration.
+![Configure a cluster backup](/documentation/cloud/backup-schedule.png)
 
-## Self-service backups
+### Restore a backup
 
-Qdrant engine offers a snapshot API that allows to create a snapshot of a particular collection or even the whole storage.
-Please refer to the [snapshot documentation](../../concepts/snapshots/) for details.
+If you have a backup, it appears in the list of **Available Backups**. You can
+choose to restore or delete the backups of your choice.
 
-Here is how you can quickly snapshot and recover a collection:
+![Restore or delete a cluster backup](/documentation/cloud/restore-delete.png)
 
-1. Take a snapshot
-   - In case of a single node cluster, simply call the snapshot endpoint on the exposed url. 
-   - In case of a multi node cluster you’d need to take a snapshot on each node that the collection resides upon. To achieve this, you simply prepend `node-{num}-` to your cluster url and call the [snapshot endpoint](../../concepts/snapshots/#create-snapshot) on the individual hosts, starting with node 0 up to the number of nodes minus one.
-   - In the response you'll get the name of the snapshot taken.
+<!-- I think we should move this to the Snapshot page, but I'll do it later -->
+
+## Backups with a snapshot
+
+Qdrant also offers a snapshot API which allows you to create a snapshot
+of a specific collection or your entire cluster. For more information, see our
+[snapshot documentation](/documentation/concepts/snapshots/).
+
+Here is how you can snapshot and recover a collection:
+
+1. Take a snapshot:
+   - For a single node cluster, call the snapshot endpoint on the exposed URL.
+   - For a multi node cluster call a snapshot on each node of the collection. 
+     Specifically, prepend `node-{num}-` to your cluster URL.
```

Comment:
We actually display the node specific URL in the UI.

Reply:
Oh that's right! It's a little hard for me to point to this (now), as we have not documented our UI. (I presume you're referring to the dashboard at `https://<url>:6334/dashboard`. 

I've set up an internal issue to do so.